recipe.mountainlab_pytools/meta.yaml

```diff
@@ -13,22 +13,23 @@ build:
   number: 0
   script:
     - test -n "$BUILT_FROM_TOPDIR" # error if not built from mountainlab-conda directory
+    - python -m pip install vdom jp_proxy_widget
```

Comment:
Ditto comments in ephys-viz above: we can use `--no-deps` here and rely on conda to pull in the requirements